src/components/Trade/TradeForm/TradeForm.tsx

```diff
@@ -894,7 +894,7 @@ export const TradeForm = ({
                     return false;
                   return new BigNumber(
                     activeAccountTradeBalances.inBalance.balance
-                  ).gt(assetInAmount);
+                  ).gte(assetInAmount);
```

Comment:
why you cannot trade all your balance away?

Reply:
This PR should be closed as its already fixed in merged PRs